### ISSUE TITLE
Travis CI support

### ISSUE THREAD

**Issue by analytically** [capitalone/Hygieia#597]
GitHub's almost default CI is Travis CI (http://travis-ci.com/) - does Hygieia integrate with it?

**Comment by jimzucker** [comment]
yes, take a look at the details on this pull request you will see the travis build.

https://github.com/capitalone/Hygieia/pull/411

**Comment by analytically** [comment]
That doesn't seem related to Travis-CI?

**Comment by amitmawkin** [comment]
if you mean whether we have collector for Travis CI, we don't as we provide collectors for the tools we are using . But it should be very easy to write one, as our data model is very generic and not tied to any build tools..

**Comment by analytically** [comment]
Care to provide an example?

**Comment by tabladrum** [comment]
We currently do not have collectors for Travis CI. It will be good to create travis webhook though. https://docs.travis-ci.com/user/notifications/
